cocos/3d/framework/mesh-renderer.ts

```diff
@@ -245,7 +243,7 @@ export class MeshRenderer extends ModelRenderer {
      */
     @type(CCFloat)
     @tooltip('i18n:model.shadow_bias')
-    @property({ group: { id: 'DynamicShadow', name: 'DynamicShadowSettings', displayOrder: 0 } })
+    @_decorator.property({ group: { id: 'DynamicShadow', name: 'DynamicShadowSettings', displayOrder: 0 } })
```

Comment:
please use @property instead of @_decorator.property so that we can do more optimization on the decorator